Makefile

```diff
@@ -4,14 +4,15 @@ CURRENT_REV  = `git rev-parse HEAD`
 RES_VERSION  ?= $(shell cat RES_VERSION)
 NIX_TYPE     =  $(shell uname -s)
 GEMS         = aggregate_root \
-	       bounded_context \
-	       ruby_event_store \
-	       ruby_event_store-rom \
-	       rails_event_store \
-	       rails_event_store_active_record \
-	       rails_event_store_active_record-legacy \
-	       rails_event_store-browser \
-	       rails_event_store-rspec
+							 bounded_context \
```

Comment:
what's the reason behind this indentation? :)

Reply:
@antonpaisov good question. I am not sure what happened there. Will see if I can fix it.